lib/CMakeLists.txt

```diff
@@ -33,13 +33,7 @@ install(
 	DESTINATION visualizers
 )
 
-file(
```

Comment:
One of the issues is that the same file with the fixed path cannot work for 2 different layouts, the "build/editable" layout, and the "package" layout.

I moved this to the "source" tree, so I could keep the same relative position of the ``add_visualizers.cmake`` file and the ``lib.natvis`` file. Another strategy would be to have different files: one file written at "build" time, but then, do not copy that file as-is to the package folder at ``install()`` time but instead generate a new one with an updated relative position.

Reply:
This doesn't quite solve my problem, because now I have to define the `add_visualizers.cmake` script manually. Instead I would like to automate the process. That was what the CMake script `file` invocation represents. In reality the scripts to generate it are somewhat more involved. I can generate the file using Conan too if that's more appropriate, but it seemed to me that I probably shouldn't generate files in `package_info`.

Reply:
I guessed that the process could be more complicated, this is why I suggested the above. The main issue is that the relative path between the ``add_visualizers.cmake`` and the ``nat.vis`` file should be correct in both places, and this cannot happen if you use exactly the same file copied at ``install()`` time. So the automated solution, if you want to support both the ``create`` and the ``editable`` different layouts, would be to make sure that 2 different files are generated:
- One ``add_visualizers.cmake`` that you create at ``build()`` time, with the relative path in the "build" or editable layout
- Another ``add_visualizers.cmake`` that you create at ``package()`` time, with the relative path of the "package" layout
- It is not necessary to create the files in ``package_info()`` only in ``build()`` and ``package()``
- If it is easier, the ``add_visualizers.cmake`` could be "patched" at ``package()`` time, doing a copy to the package folder, then patching the relative paths (it could be already an easy to patch variable that is reused in the script)

Please let me know if this helps.